src/Schema/Blueprint.php

```diff
@@ -296,6 +315,57 @@ public function dropRowDeletionPolicy(): Fluent
         return $this->addCommand('dropRowDeletionPolicy');
     }
 
+    /**
+     * @param string $name
+     * @return SequenceDefinition
+     */
+    public function createSequence(string $name): Fluent
+    {
+        $definition = new SequenceDefinition($name);
+        $this->addCommand('createSequence', ['definition' => $definition, 'ifNotExists' => false]);
+        return $definition;
+    }
+
+    /**
+     * @param string $name
+     * @return SequenceDefinition
+     */
+    public function createSequenceIfNotExist(string $name): SequenceDefinition
```

Comment:
This method is the only return type of `SequenceDefinition`.  
Is this return type correct as intended?

Reply:
This is no longer the case, all except dropXxx return SequenceDefinition now.